### ISSUE TITLE
Installed plugin through package-control version is buggy

### ISSUE THREAD

**Issue by brupelo** [SublimeLinter/SublimeLinter-pydocstyle#16]
@aparajita Hi, today I've decided to give it a shot to your linter and after being sure pydocstyle was properly installed and available to Sublime Text at startup I wasn't understanding why SublimeLinter would give me over and over the error `WARNING: pydocstyle deactivated, cannot locate 'pydocstyle@python'` :P 

I was looking at the github source code and I wasn't seeing any obvious error.

So I've decided to inspect the content of the package installed on sublime packages folder and I've seen the content wasn't matching the github repo.

After removing the installed package on sublime and then cloning directly the repo in the packages folder the linter started working.

Summing up, could you please make sure the content installed through package control matches the repo so we,  the consumers :) , can enjoy of a properly working plugin?

Thanks in advance.

**Comment by aparajita** [comment]
@brupelo I am no longer involved with SublimeLinter, try @groteworld.

**Comment by groteworld** [comment]
should be fixed with v3.1.0
